js/modules/k6/ws/ws_test.go

```diff
@@ -97,7 +97,12 @@ func TestSession(t *testing.T) {
 		Group:  root,
 		Dialer: tb.Dialer,
 		Options: lib.Options{
-			SystemTags: lib.GetTagSet("url", "proto", "status", "subproto"),
+			SystemTags: stats.ToSystemTagSet([]string{
```

Comment:
This should probably be simply `SystemTags: stats.TagURL | stats.TagProto | stats.TagStatus | stats.TagSubProto`

Reply:
As the option requires a pointer, the simple expression like above cannot work. Here I think we should keep the same functionality as previous lib.TagSet, to make sure the refactoring won't introduce high risk for regression. We can improve the code later.

Reply:
I'm fine with keeping the same functionality, but I just don't see why we need to go through string values to construct the `SystemTags` option. A simple helper function like this would be very helpful:
 ```go
func NewSystemTagSet(tags ...SystemTagSet) *SystemTagSet {
	result := new(SystemTagSet)
	for _, tag := range tags {
		result.Add(tag)
	}
	return result
}
```
and it would just allow you do do something like `SystemTags: stats.NewSystemTagSet(stats.TagURL, stats.TagProto, stats.TagStatus, stats.TagSubProto)`

Reply:
@na-- Added.